mlos_bench/mlos_bench/tests/launcher_test.py

```diff
@@ -44,16 +44,16 @@ def test_launch_main_app(root_path: str,
     with local_exec_service.temp_dir_context() as temp_dir:
 
         log_path = os.path.join(temp_dir, "mock-1shot.log")
-        (return_code, _stdout, _stderr) = local_exec_service.local_exec([
-            "./mlos_bench/mlos_bench/run.py" +
-            " --config mlos_bench/examples/mock-1shot.jsonc" +
-            f" --log_file '{log_path}'"
-        ], cwd=root_path)
+        cmd = "./mlos_bench/mlos_bench/run.py" + \
+              " --config mlos_bench/mlos_bench/tests/config/cli/mock-1shot.jsonc" + \
+              f" --log_file '{log_path}'"
+        (return_code, _stdout, _stderr) = local_exec_service.local_exec([cmd], cwd=root_path)
 
         assert return_code == 0
 
         with open(log_path, "rt", encoding="utf-8") as fh_out:
+            best_score_lines = [ln.strip() for ln in fh_out.readlines() if " INFO Env: Mock environment best score: " in ln]
             assert len([
-                ln.strip() for ln in fh_out.readlines()
-                if " INFO Env: Mock environment best score: 70.35" in ln
+                ln for ln in best_score_lines
+                if " best score: 60.0" in ln
```

Comment:
Changed some params so the score needed to change.
Expanding the logic here made debugging it easier.